### PR TITLE
Support turnip 3

### PR DESCRIPTION
**Pull request by lukaso**
Supports turnip version 3 which in turn supports gherkin 4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,11 @@ rvm:
   - 2.1.9
   - 2.2.5
   - 2.3.1
+  - 2.4.0
   - jruby-9.0.5.0
 
 gemfile:
   - Gemfile
+  - ./turnip2.gemfile
```
Comment (amar-shah) [inline .travis.yml]: Question for learning, when does travis source this file?

**Comment by amar-shah** [comment]
Looks ok to me, but I will hold of approving until CI is all passing.

File: .travis.yml (hunk 1 above)
Comment (lukaso) [inline .travis.yml]: It calls bundler with the file. However for some reason it needs a directory in the file name

**Comment by lukaso** [comment]
@amar-shah It's good to go. Give us the thumbs up!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changes
 
+## Version 2.1.6
```
Comment (amar-shah) [inline CHANGELOG.md]: You will have to bump version here also I think https://github.com/simplybusiness/rutabaga/blob/master/lib/rutabaga/version.rb

Comment (lukaso) [inline CHANGELOG.md]: See README.md which explains how to bump.

**Pull request merged**